Source/WebCore/platform/text/cocoa/LocalizedDateCache.mm

```diff
@@ -133,13 +131,14 @@ static void _localeChanged(CFNotificationCenterRef, void*, CFStringRef, const vo
 
 static float calculateMaximumWidthForWeek(const MeasureTextClient& measurer)
 {
-    std::array<float, 10> numLengths = { };
-    for (int i = 0; i < 10; i++)
+    std::array<float, 10> numLengths;
+    for (int i = 0; i < 10; ++i)
         numLengths[i] = measurer.measureText([NSString stringWithFormat:@"%d", i]);
 
-    int widestNum = std::distance(numLengths.data(), std::max_element(numLengths.data(), numLengths.data() + 10));
-    int widestNumOneThroughFour = std::distance(numLengths.data(), std::max_element(numLengths.data() + 1, numLengths.data() + 5));
-    int widestNumNonZero = std::distance(numLengths.data(), std::max_element(numLengths.data() + 1, numLengths.data() + 10));
+    std::span numLengthsSpan { numLengths };
+    int widestNum = std::distance(numLengthsSpan.begin(), std::max_element(numLengthsSpan.begin(), numLengthsSpan.end()));
```

Comment:
Can you use std::ranges for these? e.g. `std::ranges::max_element(numLengthsSpan))` rather than `std::max_element(numLengthsSpan.begin(), numLengthsSpan.end()))`?